benchopt/plotting/helpers_compat.py

```diff
@@ -55,3 +56,43 @@ def add_h_line(fig, val, xlim=None, plotly=False):
         line_dash='dot', line_color='black',
         mode='lines', showlegend=False
     ))
+
+
+def make_bars(fig, heights, ticks, width, colors, times, plotly=False):
```

Comment:
```suggestion
def _make_bars(fig, heights, ticks, width, colors, times, plotly=False):
```

I would make this function private in the API

So you have a good excuse not to write a docstring ;)